examples/morpheusvm/tests/e2e/transfer_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package e2e_test
+
+import (
+	"context"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/hypersdk/api/jsonrpc"
+	"github.com/ava-labs/hypersdk/auth"
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/crypto/ed25519"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/actions"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/vm"
+	"github.com/ava-labs/hypersdk/tests/workload"
+
+	he2e "github.com/ava-labs/hypersdk/tests/e2e"
```

Comment:
I don't know how you say this in your head, but I say `he-two-e`... it sounds great haha

Reply:
I wasn't that creative here.. I copied that from the e2e.go file..